src/adapters/postgres/fields-adapter.ts

```
@@ -2,304 +1,1773 @@
+      });
+      const response = await this.fieldsValuesRepository.update(
+        id,
+        fieldValuesUpdateDto
+      );
+
+      return response;
+    } catch (e) {
+      LoggerUtil.error(
+        `${API_RESPONSES.SERVER_ERROR}`,
+        `Error: ${e.message}`,
+      )
```

ESLint

[error] Replace `⏎········`${API_RESPONSES.SERVER_ERROR}`,⏎········`Error:·${e.message}`,⏎······)` with ``${API_RESPONSES.SERVER_ERROR}`,·`Error:·${e.message}`);`

(prettier/prettier)